Expensify/App
# remove notification Subheader

### Details

Remove notification Subheader

### Fixed Issues

$ https://github.com/Expensify/App/issues/7670

### Tests | QA Steps

 1. Click "+"
 2. Click "New room"
 3. Add any room name, choose any workspace, choose any visibility setting - Click "Create room"
 4. Click the room header in the main chat pane to be taken to the room "Setting" pane
 5.  Verify notification Subheader was removed.


### Tested On


### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/56851391/153449729-a9794fd3-260b-4827-980a-21e21b27a33f.png)


#### Mobile Web

![image](https://user-images.githubusercontent.com/56851391/153452559-7ed501c6-f1ef-4ff5-99dd-5c2bb1938056.png)

#### Desktop

![Captura de Tela 2022-02-10 às 21 33 57](https://user-images.githubusercontent.com/56851391/153520158-107e210a-eb28-442a-9821-aca5f03f7e11.png)
 

#### iOS

![Captura de Tela 2022-02-10 às 14 14 40](https://user-images.githubusercontent.com/56851391/153511717-46a39283-9d42-4f33-8859-9a6be667b366.png)

#### Android

![image](https://user-images.githubusercontent.com/56851391/153451759-f60d8607-8863-4b66-bd38-6f443311181b.png)
